events/events.go

```diff
@@ -80,6 +80,7 @@ func EmitPoetWaitRound(current, publish types.EpochID, wait time.Duration) {
 			Current: current.Uint32(),
 			Publish: publish.Uint32(),
 			Wait:    durationpb.New(wait),
+			Until:   timestamppb.New(time.Now().UTC().Add(wait)),
```

Comment:
Instead of passing in `wait` as a `time.Duration`, consider passing it in as a `time.Time` and calculate `wait` from it. The only caller of `Empit.PoetWaitRound` passes `time.Until(wait)` as `wait` parameter and could just pass `wait` instead.

Reply:
Thanks, I'll do that.